pyproject.toml

```diff
@@ -1,90 +1,188 @@
-    "nvdiffrast/torch/*.h",
-    "nvdiffrast/torch/*.inl",
-    "nvdiffrast/torch/*.cpp",
-    "nvdiffrast/tensorflow/*.cu",
-    "nvdiffrast/jax/*.h",
-    "nvdiffrast/jax/*.inl",
-    "nvdiffrast/jax/*.cpp",
+[tool.pixi.pypi-dependencies]
+b3d = { path = ".", editable = true }
+
+[tool.pixi.target.osx-arm64]
+build-dependencies = { scipy = { version = "1.14.0.*" }, numpy = { version = "1.26.4.*" } }
+
+[tool.pixi.environments]
+cpu = { features = ["cpu", "dev"], solve-group = "default" }
+gpu = { features = ["cuda", "dev"], solve-group = "cuda" }
+
+[tool.pixi.tasks.rerun]
+description = "start rerun"
+cmd = "./task-rerun.sh"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-auth]
+description = "authenticate to gcloud"
+cmd = "gcloud auth login --update-adc"
+
+[tool.pixi.tasks.gcp-list]
+description = "list gcp instances"
+cmd = "./gcp.sh :gcp-list"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-active-host]
+description = "active gcp host"
+cmd = "./gcp.sh :gcp-active-host"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-create]
+description = "create, configure, bootstrap gcp instance"
+depends_on = [
+    "gcp-create-new",
+    "gcp-config-ssh",
+    "gcp-remote-forward",
+    "gcp-bootstrap",
 ]
 
-[tool.pytest.ini_options]
-testpaths = ["tests"]
+[tool.pixi.tasks.gcp-create-new]
+description = "create gcp instance"
+cmd = "./gcp.sh :gcp-create"
+cwd = "scripts"
 
-[tool.pyright]
-typeCheckingMode = "standard"
+[tool.pixi.tasks.gcp-bootstrap]
+description = "bootstrap a gcp instance"
+cmd = "./gcp.sh :gcp-bootstrap"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-config-ssh]
+description = "ssh-config gcp instance"
+cmd = "./gcp.sh :gcp-config-ssh"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-remote-forward]
+description = "remote-forward gcp instance for rerun"
+cmd = "./gcp.sh :gcp-remote-forward"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-delete]
+description = "delete gcp instance"
+cmd = "./gcp.sh :gcp-delete"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-start]
+description = "start gcp instance"
+cmd = "./gcp.sh :gcp-start"
+cwd = "scripts"
+depends-on = ["gcp-bootstrap"]
+
+[tool.pixi.tasks.gcp-status]
+description = "status of gcp instance"
+cmd = "./gcp.sh :gcp-status"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-stop]
+description = "stop gcp instance"
+cmd = "./gcp.sh :gcp-stop"
+cwd = "scripts"
+
+[tool.pixi.tasks.gcp-ssh]
+description = "ssh into a gcp instance"
+cmd = "./gcp.sh :gcp-ssh"
+cwd = "scripts"
+depends_on = ["rerun"]
+
+[tool.pixi.feature.cpu.target.osx-arm64.dependencies]
+jax = "*"
+jaxlib = { version = "*", build = "cpu*" }
+
+[tool.pixi.feature.dev.dependencies]
+distinctipy = ">=1.3.4,<1.4"
+imageio = "*"
+jupyter = ">=1.0.0,<1.1"
+matplotlib = ">=3.9.1,<3.10"
+natsort = ">=8.4.0,<8.5"
+numpy = "<2.0.0"
+rerun-sdk = "==0.16.1"
+ruff = "*"
+trimesh = "==4.2.4"
+tqdm = "==4.66.2"
+pdoc3 = ">=0.11.1,<0.12"
+pillow = "==10.3.0"
+polars = ">=0.20,<0.21"
+pytest = "*"
+scikit-learn = ">=1.5.1,<1.6"
+scipy = ">=1.14.0,<1.15"
+
+[tool.pixi.feature.cuda]
```

Comment:
here's the cuda feature. note that it has the `system-requirement` which means an nvidia GPU with drivers installed.